randomizer/RandomizerExpNames.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+public static class RandomizerExpNames {
+
+	public static string FileName = "ExpNames.txt";
+
+	public static List<String> DefaultList = new List<String> { "Apples",  "Bananas",  "Bells",  "Bits",  "Bolts",  "Boonbucks",  "Boxings",  "Brick",  "Brownie Points", 
+			"Bytes",  "Cash",  "Coins",  "Comments",  "Credits",  "Crowns",  "Diamonds",  "Dollars",  "Dollerydoos",  "Doubloons",  "Drams",  "EXP",  "Echoes",  "Emeralds",  "Euros",
+			"Exalted Orbs",  "Experience",  "Farthings",  "Fish",  "Fun",  "GP",  "Gallons",  "Geo",  "Gil",  "Glod",  "Gold",  "Hryvnia",  "Hugs",  "Kalganids",  "Leaves",  "Likes",
+			"Marbles",  "Minerals",  "Money",  "Munny",  "Nobles",  "Notes",  "Nuts",  "Nuyen",  "Ori Money",  "Pesos",  "Pieces of Eight",  "Points",  "Poké",  "Pons",  "Pounds Sterling", 
+			"Quatloos",  "Quills",  "Rings",  "Rubies",  "Runes",  "Rupees",  "Sapphires",  "Sheep",  "Shillings",  "Silver",  "Slivers",  "Socks",  "Solari",  "Souls",  "Sovereigns", 
+			"Spheres",  "Spirit Bucks",  "Spirit Light",  "Stamps",  "Stonks",  "Strawberries",  "Subs",  "Tickets",  "Tokens",  "Vespine Gas",  "Wheat",  "Widgets",  "Wood",  "XP",
+			"Yen",  "Zenny",  "Zloty"};
+
+	public static List<String> ExpNames;
+
+	public static void ParseExpNames() {
+		if (!File.Exists(FileName)) {
+			WriteDefaultFile();
+		}
+
+		try {
+			ExpNames = new List<string>(File.ReadAllLines(FileName));
+			if (ExpNames.Count == 0) {
+				ExpNames = new List<String> {""};
```

Comment:
If there's an empty file, I think it'd be best to either fall back to `DefaultList` or always return `Experience`. If I read this correctly, the current behavior will be that all exp pickups output as `"125 "` for example.